src/options/index.ts

```diff
@@ -1,14 +1,14 @@
 import browser from "webextension-polyfill";
 
 const options = document.querySelectorAll(
-  'input[type="checkbox"]'
+  'input[type="checkbox"]',
 ) as NodeListOf<HTMLInputElement>;
 
 const urlForm = document.querySelector("#block-list-form") as HTMLFormElement;
 const urlInput = document.querySelector("#url-input") as HTMLInputElement;
 const sendButton = document.querySelector("#send-button") as HTMLButtonElement;
 const removeAllButton = document.querySelector(
-  "#remove-all-button"
+  "#remove-all-button",
```

Comment:
I think the ```,``` can be removed

Reply:
Removed! This was automatically set when I ran the `npx prettier --write .` command to fix all formatting problems, but it's not mandatory. I don't think the project has any configuration for that yet.

Reply:
Yes, and it's a good idea to add a prettier file in the future, to ensure code style.